src/Program.cs

```diff
@@ -249,6 +258,68 @@ private static void InstallBase64Certificate(string base64, string password, Sto
             }
 
             AddToStore(cert, storeName, storeLocation);
+        }
+
+        private static void FindByThumbprint(string thumbprint, StoreName storeName, StoreLocation storeLocation)
+        {
+            var store = new X509Store(storeName, storeLocation);
+            store.Open(OpenFlags.ReadOnly);
+
+            if (store.Certificates.Count == 0)
+            {
+                return;
+            }
+
+            Console.WriteLine(
+                $"Searching for certificate thumbprint '{thumbprint}' in '{storeName}/{storeLocation}'...");
+            Console.WriteLine();
+
+            bool isFound = false;
+            foreach (var certificate in store.Certificates)
+            {
+                if (certificate.Thumbprint == null ||
+                    string.Compare(certificate.Thumbprint, thumbprint,
+                        StringComparison.OrdinalIgnoreCase) != 0)
+                {
+                    continue;
+                }
+
+                isFound = true;
+                var certificateInfo = new Dictionary<string, string>
```

Comment:
Duplicated code from `ListCertificates`, it's better to extract this code into a method.